packages/create-catalyst/src/index.ts

```diff
@@ -8,6 +8,18 @@ import PACKAGE_INFO from '../package.json';
 import { create } from './commands/create.js';
 import { init } from './commands/init.js';
 
+const currentNodeVersion = process.versions.node;
+const minimumNodeVersion = 18;
+
+if (parseInt(process.versions.node.split('.')[0], 10) < minimumNodeVersion) {
+  console.error(
+    chalk.red(
+      `\nYou are running Node.js v${currentNodeVersion}. Catalyst requires Node.js v${minimumNodeVersion}.0.0 or higher.\n`,
```

Comment:
Open to wording suggestions

Reply:
Maybe include a next step, like "If you use nvm, you can run ..."